UserService/app/src/main/java/com/bookmyshow/userservice/dto/UserRequestDTO.java

```diff
@@ -0,0 +1,63 @@
+package com.bookmyshow.userservice.dto;
+
+import javax.validation.constraints.NotBlank;
+import java.io.Serializable;
+
+public class UserRequestDTO implements Serializable {
+
+    @NotBlank
+    private String firstName;
+
+    @NotBlank
+    private String lastName;
+
+    @NotBlank
+    private String email;
+    private Long roleId;
```

Comment:
Fields are redundant in UserRequestDTO and UserResponseDTO. Can't we have only one UserDTO to manage both request and response?

Reply:
we don't have an id in the request.
previously I was trying out different DTOs to represent the relationship between role and permission.